_docs_integrate/migration-from-v4-to-v5.md

```diff
@@ -72,46 +73,58 @@ The aspects to be taken into account when migrating to version 5, which were bri
+It fulfills the same role as the previous `request.content` property of the RelationshipChange with the difference that it now has the type [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) instead of RelationshipCreationChangeRequestContent if a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) whose `content` is of the type [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) was used for the initial creation of the pending Relationship.
+Otherwise, the type of the `creationContent` property of the Relationship is [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent).
+Before the introduction of the [new mandatory wrappers for non-standard content]({% link _docs_integrate/migration-from-v4-to-v5.md %}#new-mandatory-wrappers-for-non-standard-content-of-messages-relationshiptemplates-and-relationships), such arbitrary `creationContent` of the Relationship was stored in the `request.content` property of the RelationshipChange by also permitting the storage of data of unknown type.
+
+The functionalities of the old two use cases of accepting and rejecting RelationshipChanges are now provided by the two new use cases [Accept Relationship]({% link _docs_use-cases/use-case-transport-accept-relationship.md %}) and [Reject Relationship]({% link _docs_use-cases/use-case-transport-reject-relationship.md %}). Accordingly, the Connector routes using RelationshipChanges were replaced. When using the two new Connector routes, which work directly on the Relationships, a request body no longer needs to be provided.
+This means that no `content` must be sent with any of these operations anymore.
+The former, more generic Connector routes using RelationshipChanges could also have been used to accept or reject changes to a Relationship other than the establishment of an active Relationship, which might have required the transmission of additional data.
 
 | Old Connector route                                                                | New Connector route                                           |
 | ---------------------------------------------------------------------------------- | ------------------------------------------------------------- |
 | `PUT /api/v2/Relationships/{relationshipId}/Changes/{changeId}/Accept` (with body) | `PUT /api/v2/Relationships/{relationshipId}/Accept` (no body) |
 | `PUT /api/v2/Relationships/{relationshipId}/Changes/{changeId}/Reject` (with body) | `PUT /api/v2/Relationships/{relationshipId}/Reject` (no body) |
 
+However, please note that the Connector route `POST /api/v2/Relationships` for [creating a Relationship with a RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-relationship-with-relationshiptemplate.md %}) is unchanged.
```

Comment:
do we have to talk about unchanged things in a migration guide? IMO this rather distracts from the main content

Reply:
I agree, I will remove it.